### ISSUE TITLE
Tooltips for metrics

### ISSUE THREAD

**Issue by ing200086** [phpmetrics/PhpMetrics#292]
I really like phpmetrics. I have a recommendation for an enhancement about information about the metrics. For each metric in the table, an information icon tooltip which has a short description of the metric and information about poor/average/good/perfect scores.

I know that I can go to the documentation page, but those short tooltips would be a nice touch.

**Comment by niconoe-** [comment]
Thank you for your feedback :).

Can you give us examples of where in the generated website you want to add some informaion?

BTW, I don't think it should be tooltips: these are not (or hardly) accessible. Perhaps we can just always display the metrics informations...

**Comment by ing200086** [comment]
Thanks for responding.

I am really just trying to recommend a way to have the columns for the different report sections. Be more accessible in plain English. What would you think about having a collapsible panel somewhere on each report page which provides a clearer explanation for each number?

Some interesting items for each would be an explanation for each number, function output boundaries, old values & relative % change, recommended methodologies to lower/raise each score.

These are just ideas, not requirements :)

**Comment by nickescobedo** [comment]
Started using this recently and came to say this exact thing. Perhaps a ? that someone can hover showing the definition. Not opposed to something else but remembering all the acronyms is very difficult.
